src/Home/Home.tsx

```diff
@@ -1,5 +1,93 @@
 import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import glamorous from 'glamorous';
+// const Home = () => <h1>Home Page</h1>;
+const { Div } = glamorous;
 
-const Home = () => <h1>Home Page</h1>;
+const CircleButton = glamorous.button({
+  borderRadius: '50%',
+  backgroundColor: 'black',
+  width: '4vw',
+  height: '4vw',
+  padding: 3,
+});
+
+const RectangleThing = glamorous.div({
```

Comment:
Let's move this component into a separate file. I think maybe a more descriptive name than `RectangleThing` might also be a good idea--perhaps `PageLink`, since that's what each rectangle will contain?